packages/map/dev/reactiveMapEntries.tsx

```diff
@@ -0,0 +1,101 @@
+// changes to this file might be applicable to similar files - grep 95DB7339-BB2A-4F06-A34A-25DDF8BF7AF7
+
+import { createStore } from "solid-js/store";
+import { createEffect } from "solid-js";
+import { MapEntries } from "../src/index.js";
+import { TransitionGroup } from "solid-transition-group";
+import { ReactiveMap } from "@solid-primitives/map";
+
+const foods = [
+  "oatmeal",
+  "plantains",
+  "cranberries",
+  "chickpeas",
+  "tofu",
+  "Parmesan cheese",
+  "amaretto",
+  "sunflower seeds",
+  "grapes",
+  "vegemite",
+  "pasta",
+  "cider",
+  "chicken",
+  "pinto beans",
+  "bok choy",
+  "sweet peppers",
+  "Cappuccino Latte",
+  "corn",
+  "broccoli",
+  "brussels sprouts",
+  "bread",
+  "milk",
+  "honey",
+  "chips",
+  "cookie",
+];
+const randomIndex = (list: readonly any[]): number => Math.floor(Math.random() * list.length);
+const getRandomFood = () => foods[randomIndex(foods)]!;
+const randomKey = (map: ReactiveMap<string, string>): string => {
+  const keys = Array.from(map.keys());
+  return keys[randomIndex(keys)]!;
+};
+
+export default function App() {
+  const [store, setStore] = createStore<{ entries: ReactiveMap<string, string> }>({
+    entries: new ReactiveMap([
```

Comment:
This could just be a signal
using a store for only one field is kinda an overkill